src/device/device_default.cpp

```diff
@@ -263,6 +263,11 @@ namespace hw {
             return true;
         }
 
+        bool device_default::derive_view_tag(const crypto::key_derivation &derivation, const std::size_t output_index, crypto::view_tag &view_tag) {
+            crypto::derive_view_tag(derivation, output_index, view_tag);
+            return true;
+        }
+
```

Comment:
Just a question that popped up in my mind: Do hardware wallets need now modifications in their tx building code as well, to compute and include view tags?

Reply:
Question already answered: A little further down I saw that you already updated the Ledger code yourself.

Reply:
Would definitely be nice to test on hardware wallets if someone has them available for testing